docs/src/pom.md

```diff
@@ -50,6 +168,24 @@ class SearchPage {
 module.exports = { SearchPage };
 ```
 
+Page objects can then be used inside a test.
+
+```js
+// search.spec.js
+const { SearchPage } = require('./models/Search');
+
+// In the test
+const page = await browser.newPage();
+const searchPage = new SearchPage(page);
+await searchPage.navigate();
+await searchPage.search('search query');
+```
+
+## Implementation
```

Comment:
We should just port snippets above to all the languages and use the same content.